### PR TITLE
[WIP]Chinese

### PR DESCRIPTION
**Pull request by 374365283**
Hello，l have translated part of the OOD UI into Chinese.
This is the progress of my work.

### CODE REVIEW

File: apps/dashboard/config/locales/zh-CN.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+#
+# This would use the information in config/locales/es.yml.
+#
+# To learn more, please read the Rails Internationalization guide
+# available at http://guides.rubyonrails.org/i18n.html.
+
+en:
+  dashboard:
+    # Site specific translations
+    batch_connect_sessions_status_bad: "您的会话已进入不良状态。 请随时与支持人员联系以获取更多信息。"
```
Comment (summerwang) [inline apps/dashboard/config/locales/zh-CN.yml:25]: 你的工作进程出现问题。请与支持人员联系以获取更多信息。

**Comment by ericfranz** [comment]
@summerwang could you review once more and either add more comments or approve?

File: apps/dashboard/config/locales/zh-CN.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+zh-CN:
+  dashboard:
+    # Site specific translations
+    batch_connect_sessions_status_bad: "你的会话出现问题。请与支持人员联系以获取更多信息。"
+    batch_connect_sessions_status_missing_connection: |
+      此应用缺少建立连接所需的信息。 如果看到此消息，请与支持人员联系。
+    batch_connect_sessions_status_queued: |
+      由于您的工作目前正在排队，请耐心等待。 等待时间取决于内核数以及请求的时间。
+    nav_help_change_password: "修改HPC密码"
+    nav_help_docs: "在线文档"
```
Comment (summerwang) [inline apps/dashboard/config/locales/zh-CN.yml:31]: 帮助文档

File: apps/dashboard/config/locales/zh-CN.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    nav_help_support: "联系支持人员"
+    nav_help_two_factor: "配置双重身份验证"
+
+    quota_additional_message: "考虑删除或归档文件以释放磁盘空间。"
+    quota_reload_message: "重新加载页面以查看更新的配额。 配额每5分钟更新一次。"
+    quota_warning_prefix_html: "配额限制警告"
+
+    balance_additional_message: "考虑请求额外的 %{balanace_units}"
```
Comment (summerwang) [inline apps/dashboard/config/locales/zh-CN.yml:39]: 考虑申请额外的 %{balanace_units}

File: apps/dashboard/config/locales/zh-CN.yml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      <pre>%{exception_trace}</pre>
+    apps_system_apps_title: "系统应用"
+
+    batch_connect_form_data_root: "数据根目录"
+    batch_connect_form_launch: "启动"
+    batch_connect_form_reset_resolution: "重置分辨率"
+    batch_connect_form_session_data_html: |
+      * 你可以通过%{data_link_tag}来获取会话数据%{title}。
+    batch_connect_no_apps: "没有可用的互动应用程序。"
+    batch_connect_no_sessions: "您没有活动的会话。"
```
Comment (summerwang) [inline apps/dashboard/config/locales/zh-CN.yml:69]: 您没有可运行的会话

File: apps/dashboard/config/locales/zh-CN.yml [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    batch_connect_sandbox: " [Sandbox]"
+    batch_connect_sessions_data_html: |
+      你可以通过%{data_link_tag}来获取会话数据%{title}。
+    batch_connect_sessions_error_invalid_job_name_html: |
+      如果由于作业名称无效而导致该作业提交失败，请要求管理员配置OnDemand来设置环境变量OOD_JOB_NAME_ILLEGAL_CHARS。
+    batch_connect_sessions_delete_confirm: "你确定吗？"
+    batch_connect_sessions_delete_hover: "删除会话"
+    batch_connect_sessions_delete_title: "删除"
+    batch_connect_sessions_errors_staging: "无法登台模板，并显示以下错误："
```
Comment (summerwang) [inline apps/dashboard/config/locales/zh-CN.yml:78]: 无法调用模板，并显示以下错误：

**Comment by 374365283** [comment]
@ericfranz could you please check whether my work can be merged .

**Comment by ericfranz** [comment]
@374365283 Let me double check with @summerwang for one final pass but I'll try to merge this week.

**Comment by ericfranz** [comment]
@374365283 we are building release notes for 1.8 and would like to credit you for this work. We could say something like:

>Special thanks to of https://github.com/374365283 and https://github.com/flybirdkh of Shanghai Jiao Tong University https://hpc.sjtu.edu.cn/ for this work

If you want us to credit you with names instead of GitHub handles or want us to credit you differently just share here.

**Pull request merged**